### PR TITLE
PBI-7-8: Teacher & Student Can See People In Course

### PR DESCRIPTION
**Pull request by IlmaMannix**

### CODE REVIEW

File: .github/workflows/sonarqube.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - name: SonarQube Scan
+      uses: sonarsource/sonarqube-scan-action@master
+      env:
+          SONAR_TOKEN: "69f012beb8463b6cfd65378dcfde0d89e000cbea"
+          SONAR_HOST_URL: "http://35.224.113.103:9000"
```
Comment (nofamex) [inline .github/workflows/sonarqube.yml]: please do not commit this again, its already exist by default on any branch

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=badaso-lms-module
+sonar.sources=src/
```
Comment (nofamex) [inline sonar-project.properties]: also remove this, its already exist

File: src/Commands/BadasoLMSSetup.php [hunk 3]
```diff
@@ -35,23 +37,36 @@ public function __construct() {
...
-    protected function addingBadasoEnv() {}
+    protected function addingBadasoEnv()
+    {
+    }
 
-    protected function addPostTablesToHiddenTables() {}
+    protected function addPostTablesToHiddenTables()
+    {
+    }
 }
```
Comment (nofamex) [inline src/Commands/BadasoLMSSetup.php]: please pull the latest from staging

**Review by nofamex — CHANGES_REQUESTED** [review]
kindly checked previously existing file on the staging branch

File: src/Controllers/LMSController.php [hunk 4]
```diff
@@ -4,4 +4,6 @@
 
 use Uasoft\Badaso\Controllers\Controller;
 
-class LMSController extends Controller {}
+class LMSController extends Controller
+{
+}
```
Comment (AldiNFitrah) [inline src/Controllers/LMSController.php]: I personally prefer the old version, but it's okay

File: tests/Feature/PeopleInClassTest.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    public function testTeacherIsExist()
+    {
+        $response = CallHelperTest::withAuthorizeBearer($this)->json('GET', CallHelperTest::getUrlApiV1Prefix("/role/{$TEACHER_ID}"));
+        $this->assertEmpty($response);
+    }
+
+    public function testStudentIsExist()
+    {
+        $response = CallHelperTest::withAuthorizeBearer($this)->json('GET', CallHelperTest::getUrlApiV1Prefix("/role/{$STUDENT_ID}"));
+        $this->assertEmpty($response);
```
Comment (Zsaschz) [inline tests/Feature/PeopleInClassTest.php]: roses are red violet are blue,
this should be $response->assertSuccesfull() uwu

**Review by AldiNFitrah — APPROVED** [review]
other than the other reviewers have mentioned, LGTM!

File: src/BadasoLMSModule.php [hunk 6]
```diff
@@ -2,12 +2,12 @@
...
+class BadasoLMSModule
+{
     protected $protected_tables = [];
 
     public function getProtectedTables(): array
     {
         return $this->protected_tables;
     }
 }
-
```
Comment (fxdros) [inline src/BadasoLMSModule.php]: Please pull the latest from the `staging` branch

File: src/Facades/BadasoLMSModule.php [hunk 7]
```diff
@@ -4,4 +4,6 @@
 
 use Illuminate\Support\Facades\Facade;
 
-class BadasoLMSModule extends Facade {}
+class BadasoLMSModule extends Facade
+{
+}
```
Comment (fxdros) [inline src/Facades/BadasoLMSModule.php]: Please pull the latest from the `staging` branch

File: src/Providers/BadasoLMSModuleProvider.php [hunk 8]
```diff
@@ -4,4 +4,6 @@
 
 use Illuminate\Support\ServiceProvider;
 
-class BadasoLMSModuleProvider extends ServiceProvider {}
+class BadasoLMSModuleProvider extends ServiceProvider
+{
+}
```
Comment (FXDROS) [inline src/Providers/BadasoLMSModuleProvider.php:9]: Please pull the latest from the `staging` branch

Comment (darren-ngoh) [inline src/Providers/BadasoLMSModuleProvider.php]: Please pull the latest from the staging branch

File: src/Seeder/BadasoLMSModuleSeeder.php [hunk 9]
```diff
@@ -4,4 +4,6 @@
 
 use Illuminate\Database\Seeder;
 
-class BadasoLMSModuleSeeder extends Seeder {}
+class BadasoLMSModuleSeeder extends Seeder
+{
+}
```
Comment (darren-ngoh) [inline src/Seeder/BadasoLMSModuleSeeder.php]: Please pull the latest from the staging branch

**Comment by Zsaschz** [comment]
please make this pr WIP if you are still going to work on it

**Review by nofamex — APPROVED** [review]

**Review by AldiNFitrah — APPROVED** [review]
LGTM already, thanks!

**Pull request merged**